imap_processing/cdf/imap_cdf_manager.py

```diff
@@ -11,11 +11,28 @@
 
 
 class ImapCdfAttributes(CdfAttributeManager):
-    """Contains IMAP specific tools and settings for CDF management."""
+    """
```

Comment:
Looks like this PR includes some stuff from your test PR - that will go away once you merge the test PR.